superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -125,6 +126,42 @@ function dbReducer(
 
 const DEFAULT_TAB_KEY = '1';
 
+const StyledDBModal = styled(StyledModal)`
```

Comment:
can you move these styles to the styles file?

Reply:
Any reason why you don't want to update StyledModal instead of creating a new style?